source/_posts/2015-04-28-open-sourcing-your-apps.markdown

```diff
@@ -0,0 +1,63 @@
+</center>
+
+
+Let's go over the process we went through to get to that point.
+
+<!-- more -->
+
+Credit where credit is due, when we were [working on Eidolon](/blog/2014/11/13/eidolon-retrospective/), our CTO [dB](http://code.dblock.org/) just casually tossed the idea that, really, Eigen should be open source too. This totally threw me for a loop, we were only just getting to a point where we could build an app from scratch in the open. Trying to get a project that had existed for years and had it's own momentum converted would take a lot of thinking about.
+
+We devoted time at the end of 2014 to understand what the constraints were for getting the app opened. From a purely functional perspective we would have to start with a [drastic decision](https://github.com/artsy/mobile/issues/11) around the repo itself. 
+
+### The Repo
+
+We opted to go for a total repo switch, removing all history. There were a lot of places where keys could have been hiding within the app. To get this done we migrated the existing `eigen` to `eigen-private` on github, and did a fresh `git init`. People who have joined Artsy Mobile post-OSS have never even cloned the `eigen-private` repo. So I feel good about this call.
+
+We used the last few moments of the private repo to remove all of the default Apple copyright notices. We didn't feel they added anything on top of the git history, and make it feel like the founders of a project are more important than anyone working on improvements.
+
+It wasn't all smooth sailing with respect to the repo switch however. As the switch happened the WatchKit came out, and we had devoted quite a lot of time to building an app in it. Given that you [can't predict Apple](http://www.elischiff.com/blog/2015/3/24/fear-of-apple)'s reactions, we opted to work on a branch from our private repo. [For months](https://github.com/artsy/eigen/pull/302). In the end it was easier to have the two folders next to each other, then copy & paste over all the files and to set all the settings in the Xcproject again.
+
+### Docs
+
+We had to significantly update our README, and a lot of the process around bootstrapping. We wanted to reduce the friction to actually trying the app as much as possible. It's easy to look at the source on github but to be able to get it up and running quickly should be a really high priority. So our README is based on getting it up and running as an OSS project, not for someone internal who may push betas/releases. 
+
+When we opened the repo, there was a lot of documentation fix PRs. They're low barriers to entry, and fun to make for people on big projects. We still get them pretty regularly.
```

Comment:
there _were_

It makes more sense for something to _have_ a low barrier to entry that to _be_ that low barrier imho